glide.lock

```diff
@@ -1,5 +1,5 @@
-hash: 0cc412c8ccbe6bfefac4bc281cef4d2836b1845b7e1fd9b17497229851380eaa
```

Comment:
the lock file change should be in the bump commit

Reply:
commit: `bump(*): kubernetes 1.17-rc.2`
files: `glide.yaml` and `glide.lock`

commit: `update vendor folder`

commit: fix `make verify`

Is this what you are referring to @sttts ?

Reply:
Would just do two commits:

- bump(*): kubernetes 1.17-rc.2
- update generated files

Alternatively, sometimes easier to review:

- Pin kube dependencies to 1.17-rc.2
- bump(*)
- update generated files

In both cases the glide.lock should be in the same commit vendor is bumped.

Reply:
done, please take a look when you have time @sttts